### DISCUSSION TITLE
I believe it's possible this app is no longer safe

### DISCUSSION THREAD

**Discussion by reconrad48** [lencx/ChatGPT#849] [General]
I just found out that the API that I used for this desktop app was hacked and someone made racked up my bill to $120, which fortunately I placed a cap on.. this app has not been updated, and earlier today I saw that a regular size version was open ( I was using) but in the system tray, there was a mini-version running in the background... plus, the app would no longer let me sign out... GET THIS OFF YOUR MACHINE and change your OpenAI password ASAP, and revoke all your keys.

**Comment by banch** [comment]
Would like to hear more from this? Can you see the ip who changed you VM bills or?

**Reply by reconrad48** [reply]
I doubt it, but I did submit the request to openai to have more info on the API calls, including IP etc, but if they infiltrated the app locally on my PC, then it's possible that it will show my IP. But this app is compromised regardless.. people aren't even paying attention to the fact lencx has completely abandoned this app. He has not responded in a month, and hasn't made any updates. Shame on him for not warning us and telling us to get it off our computers, although he did warn of a trojan last month, he did not tell us that our current version was compromised. Listen, this app REQUIRES OUR OPENAI LOGIN, which is not good anyway. It should only need our API key. When I first installed it back when it started, I thought it was an official OpenAI desktop app. Now I see that I was wrong, so NOTHING should ever require you to login with your credentials. He was up to no good from the start.

**Comment by PEEKPerformer** [comment]
This application is a wrapper for the web version of chatgpt. This application doesn't use the API at all.

**Comment by Drako01** [comment]
Thank You

**Comment by reconrad48** [comment]
no, worse, it has access to your login credentials.

**Reply by asheroto** [reply]
It's a web browser. The software uses Microsoft WebView2. Per their security, password fields are not able to be read except by the browser. If still concerned, you can use 2FA. :-)

**Reply by harunpoter** [reply]
no its Tauri-powered browser application interface

**Comment by wakeUPslow** [comment]
Is this app safe to use or not....?the NoFWL version has talks of concerns about it not being safe to use....

If its not safe to use, then why are so many people using it/fork it?

Any suggestions on a good/safe desktop app to utilize chatGPT or any AI on Linux?
